app/assets/javascripts/text-area-autosize.js

```diff
@@ -0,0 +1,3 @@
+$(function() {
+  $(".js-text-area-auto-size").textareaAutoSize();
```

Comment:
As I mentioned with the image uploading PR, this could be re-written slightly as a module: https://github.com/alphagov/govuk_admin_template/blob/master/JAVASCRIPT.md

Even though this is a jquery plugin, should we have some javascript unit tests?